webpack/assets/javascripts/react_app/components/notifications/notifications.test.js

```diff
@@ -35,7 +35,7 @@ describe('notifications', () => {
     };
   });
 
-  it('full flow', () => {
+  xit('full flow', () => {
```

Comment:
Thanks @sharvit @tbrisker,
for some reason it makes the notification test heap out of memory,
I didn't spot any regression in the browser or in redux devtools..

@johnpmitsch did you encounter something like that? did using r-t-lib solved it for you?

Reply:
I'm not sure about the notification test oom errors, I haven't seen anything like that, r-t-lib would be a different test so could solve the issue by testing in a different way, but it would mean the test would need  to be re-written.